.github/workflows/kind-e2e.yaml

```diff
@@ -87,6 +116,7 @@ jobs:
       NODE_SHA: ${{ matrix.kind-image-sha }}
 
       EVENTING_KAFKA_BROKER_CHANNEL_AUTH_SCENARIO: ${{ matrix.channel-auth-scenario }}
+      EVENTING_KAFKA_BROKER_CLASS: ${{ matrix.broker-class }}
```

Comment:
Eventing uses "BROKER_CLASS" can we use the same?

https://github.com/knative/eventing/blob/15dd7ca8c81146cd4ff0f5b5ad1d5fe3820df9ec/test/rekt/resources/broker/broker.go#L48-L51